.travis.yml

```diff
@@ -99,34 +102,22 @@ jobs:
 
     - name: PHP unit tests (7.4, WordPress latest, with code coverage)
       php: "7.4"
-      env: WP_VERSION=latest DEV_LIB_ONLY=phpunit,coverage
+      env: WP_VERSION=latest DEV_LIB_ONLY=phpunit,coverage,composer
       before_script:
         - echo "Running PHP unit tests with code coverage ..."
 
     - name: PHP unit tests (7.4, WordPress trunk)
       php: "7.4"
-      env: WP_VERSION=trunk DEV_LIB_ONLY=phpunit
-
-    - name: PHP unit tests (7.3, WordPress latest)
-      php: "7.3"
-      env: WP_VERSION=latest DEV_LIB_ONLY=phpunit
-
-    - name: PHP unit tests (7.2, WordPress latest)
```

Comment:
Limit these to a few defaults -- the lowest and the current version of PHP and WP.